ci_scripts/all-utils.sh

```diff
@@ -106,3 +106,13 @@ verify_bazel_rebuild()
     exit 1
   fi;
 }
+
+symlink_electron()
+{
+  ln -s Electron.app/Contents/MacOS/Electron node_modules/electron/dist/node
```

Comment:
Seems like a pretty invasive environment modification, why is this needed? Will users need to do this to use the module on Electron?

Reply:
This is something we always did, as much as I can tell users don't need but we need because electron :/

Reply:
As much as I remember, it's mostly because we are using the same code from NodeJS so our shebang relies on it.